openshift/ftest-template-standalone/src/test/java/org/arquillian/cube/openshift/standalone/HelloWorldTemplatesIT.java

```diff
@@ -0,0 +1,50 @@
+package org.arquillian.cube.openshift.standalone;
+
+import java.io.IOException;
+import java.net.URL;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import org.arquillian.cube.openshift.api.Template;
+import org.arquillian.cube.openshift.api.TemplateParameter;
+import org.arquillian.cube.openshift.api.Templates;
+import org.arquillian.cube.openshift.impl.enricher.AwaitRoute;
+import org.arquillian.cube.openshift.impl.enricher.RouteURL;
+import org.arquillian.cube.openshift.impl.requirement.RequiresOpenshift;
+import org.arquillian.cube.requirement.ArquillianConditionalRunner;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+@Category(RequiresOpenshift.class)
+@RequiresOpenshift
+@RunWith(ArquillianConditionalRunner.class)
+@Templates(templates = {
+    @Template(url = "classpath:hello-openshift-templates.yaml",
+        parameters = @TemplateParameter(name = "RESPONSE", value = "Hello from Arquillian Templates"))
+})
+public class HelloWorldTemplatesIT {
+
+    @RouteURL("hello-openshift-templates-route")
+    @AwaitRoute
+    private URL helloOpenshiftTemplates;
+
+    @Test
+    public void should_create_class_template_resources_from_second_template() throws IOException {
+        verifyResponse(helloOpenshiftTemplates);
+    }
+    private void verifyResponse(URL url) throws IOException {
```

Comment:
Why a separate method? There is a single test and shouldn't the test be more verbose in stating what we are testing.